File: 1_TwoSum/twoSum.java

public class twoSum {
    class Solution {
        public static int[] twoSum(int[] nums, int target) {
            int l = nums.length;
            
            for(int i=0; i<l; i++){
                for (int j=i+1; j<l; j++){
                    if (nums[i]+nums[j] == target){
                        return new int[] {i, j};
                    }
                }
            }
    
            return null;
        }
    }
    
    public static void main(String[] args){
        int[] nums = {2, 7, 11, 15};
        int target = 9;

        int[] ans = Solution.twoSum(nums, target);

        System.out.println("[" + ans[0] + " , " + ans[1] + "]");
    }
}

File: 1_TwoSum/TwoSum.cpp
#include<iostream>
#include<vector>

using namespace std;

class Solution {
    public:
    static vector<int> twoSum(vector<int>& nums, int target){
        int l=nums.size();

        for (int i=0; i<l; i++){
            for (int j=i+1; j<l; j++){
                if (nums[i]+nums[j] == target){
                    return {i, j};
                }
            }
        }
    }
};

int main() {

    vector<int> nums = {2, 7, 11, 15};
    int target = 9;

    vector<int> ans = Solution::twoSum(nums, target);
    
    printf("[%d , %d]", ans[0], ans[1]);

    return 0;
}